tests/integration/commands/test_commands.py

```diff
@@ -143,6 +143,7 @@ def test_get_port_props(self):
         )
         self.validate_results_df(result)
 
+    @unittest.skip("Jira: SENG-40")
```

Comment:
Same here for a bit of description, mainly so we can see in the logs why this gets skipped without having to go back and look at the jira